shared/settings/account/add-modals.tsx

```diff
@@ -222,9 +222,12 @@ export const VerifyPhone = () => {
       onClose={onClose}
       header={{
         hideBorder: true,
+        leftButton: Styles.isMobile ? (
+          <Kb.BackButton onClick={onClose} iconColor={Styles.globalColors.white} />
+        ) : null,
         style: styles.blueBackground,
         title: (
-          <Kb.Text type="BodySmall" negative={true}>
+          <Kb.Text type="BodySmall" negative={true} center={true}>
             {displayPhone || 'Unknown number'}
```

Comment:
🐑 🐑 🐑 do you want to recapitalize the N?

Reply:
Naw, I think our standard is not to capitalize.

Reply:
o I had thought your previous iteration of this PR capitalized it.

Reply:
oh I see I looked at initially before you had undone that